src/js/calculator/home_care_client_fee/home_care_client_fee.js

```diff
@@ -201,8 +201,7 @@ class HomeCareClientFee {
         },
       },
       shopping_service_prices: {
-        first_per_week: 9.37,
-        others_per_week: 11.35,
+        first_per_week: 9.37
```

Comment:
I had kept the trailing comma here (and elsewhere) to avoid extra git changes. AFAIK our preprocessor allows it.